test/dynamo/test_misc.py

```diff
@@ -852,7 +852,9 @@ def f(x, y, z, n):
                     """\
 def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3][1]cpu", arg3_1: "f32[3][1]cpu", arg4_1: "f32[3][1]cpu"):
         # No stacktrace found for following nodes
-        foo_default = torch.ops.mylib.foo.default(arg4_1, [arg2_1, arg3_1], arg1_1, 2, arg0_1);  arg4_1 = arg2_1 = arg3_1 = arg1_1 = arg0_1 = foo_default = None
+        as_strided_default: "f32[3][1]cpu" = torch.ops.aten.as_strided.default(arg4_1, [3], [1], 0);  arg4_1 = None
```

Comment:
avoid calling as_strided_default when src and dst size, stride, offset are the same for more readable code.